docs/source/tracking.rst

```diff
@@ -576,6 +549,31 @@ Artifact Stores
 In addition to local file paths, MLflow supports the following storage systems as artifact
 stores: Amazon S3, Azure Blob Storage, Google Cloud Storage, SFTP server, and NFS.
 
+The artifact store is a location suitable for large data (such as an S3 bucket or shared NFS
```

Comment:
Could you move this block above the `In addition to local file paths, MLflow supports the following storage systems as artifact stores: Amazon S3, Azure Blob Storage, Google Cloud Storage, SFTP server, and NFS.`? We should describe what the artifact store is before mentioning the supported storage systems.

Reply:
updated